### PR TITLE
further improve the docker image

### PR DESCRIPTION
**Pull request by cleverca22**

### CODE REVIEW

**Comment by erikd** [comment]
Need to apply the patch in the CI artifacts to upate the `cabal.project` file.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM ubuntu
```
Comment (disassembler) [inline Dockerfile]: ubuntu:18.04

File: docker/default.nix [hunk 2]
```diff
@@ -39,11 +48,29 @@ let
       ];
     };
   };
+  oauth_configuration = {
+    services.oauth2_proxy = {
+      redirectURL = "http://localhost.earthtools.ca/oauth2/callback";
```
Comment (disassembler) [inline docker/default.nix]: we shouldn't reference your personal domain here

File: docker/default.nix [hunk 3]
```diff
@@ -39,11 +48,29 @@ let
...
+      cookie.secure = false;
+    };
+    services.monitoring-services = {
+      oauth = {
+        enable = true;
+        emailDomain = "iohk.io";
```
Comment (disassembler) [inline docker/default.nix]: shouldn't reference iohk.io google domain either.

**Review by disassembler — APPROVED** [review]
looks great! Just some minor changes.

File: docker/default.nix [hunk 4]
```diff
@@ -302,6 +342,10 @@ let
     chmod u+x /etc/runit/stopit
     kill -cont 1
   '';
+  web-api = mkService "web-api" ''
+    export PGPASSFILE=${eval.config.services.cardano-exporter.pgpass}
+    ${self.cardano-explorer}/bin/cardano-explorer
+  '';
```
Comment (disassembler) [inline docker/default.nix:355]: this should be a service in `nix/nixos`

**Review by disassembler — CHANGES_REQUESTED** [review]

**Pull request closed without merging**